contrib/mesos/pkg/minion/server.go

```diff
@@ -204,6 +205,15 @@ func (ms *MinionServer) launchHyperkubeServer(server string, args *[]string, log
 		log.Infof("wrote %d bytes to %v", written, logFileName)
 	}()
 
+	// use given environment, but add /usr/sbin to the path for the iptables binary used in kube-proxy
```

Comment:
couple of thoughts:
- avoid two calls to `os.Environ()`; DRY
- is this worth parameterizing? for example `./km minion --ensure-path=/usr/sbin:/sbin`
  - this would append each entry if it wasn't present in the existing PATH list
  - flag would need to propagate from scheduler invocation